Branch-Xamarin-SDK/Branch.cs

```diff
@@ -40,11 +40,11 @@ public class Branch
 		int retries = -1;
 
 		/// <summary>
-		/// Gets or sets the application key. This is generally
+		/// Gets or sets the Branch key. This is generally
 		/// set by the platform specific Init methods.
 		/// </summary>
-		/// <value>The app key.</value>
-		public String AppKey { get; set; }
+		/// <value>The Branch key.</value>
+		public string BranchKey { get; set; }
```

Comment:
Shoot, I thought I reverted that :disappointed: Was trying to avoid any cosmetic changes with this PR.